Misc/NEWS.d/next/Library/2020-10-20-00-34-00.bpo-2190.WNIr6.rst

```diff
@@ -0,0 +1,4 @@
+:class:`http.cookiejar.MozillaCookieJar` will now correctly load and
+save cookies marked `HTTP-only
+<https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies>`.  PR by
```

Comment:
We usually say 'patch' rather than 'PR'.  But leave until do another change.

Reply:
@terryjreedy Anything else needed from me to get this merged?